include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1779,10 +1779,8 @@ namespace internal
               transfer.schemes[i].level_dof_indices_fine.data();
           }
 
-        bool     fine_indices_touch_non_active_dofs = false;
-        IndexSet locally_active_dofs;
-        DoFTools::extract_locally_active_dofs(dof_handler_coarse,
-                                              locally_active_dofs);
+        bool     fine_indices_touch_non_locally_owned_dofs = false;
```

Comment:
I think we can use a simpler variable name
```suggestion
        bool     fine_indices_touch_remote_dofs = false;
```